src/System.Data.Odbc/tests/System.Data.Odbc.Tests.builds

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I wasn't working 100% so I stayed with what I had. But I'll definitely cherry-pick it before merging!